### PR TITLE
Add bindings to `v8::Function::GetScript{Line,Column}Number`

### PR DESCRIPTION
**Pull request by sujayakar**
I'd like to use these methods in my application, so I added them! This is my first time contributing to `rusty_v8`, so let me know if there's anything I'm missing.

### CODE REVIEW

File: tests/test_api.rs
Comment (bartlomieju) [inline tests/test_api.rs:2027]: This assertion should use hardcoded column number

File: tests/test_api.rs
Comment (bartlomieju) [inline tests/test_api.rs:2040]: This one too

**Review by bartlomieju — COMMENTED** [review]
Thank you for PR @sujayakar! It looks mostly good I have only nitpicks to assertions in test

**Review by bartlomieju — APPROVED** [review]
LGTM, thank you @sujayakar I'll wait for another review before landing

File: src/binding.cc
Comment (bnoordhuis) [inline src/binding.cc:1570]:
```suggestion
  return ptr_to_local(&self)->GetScriptColumnNumber();
```
kLineOffsetNotFound == -1 so the ternary is somewhat superfluous.

File: src/binding.cc
Comment (bnoordhuis) [inline src/binding.cc:1575]:
```suggestion
  return ptr_to_local(&self)->GetScriptLineNumber();
```

File: src/function.rs
Comment (bnoordhuis) [inline src/function.rs:484]:
```suggestion
  pub fn get_script_column_number(&self) -> Option<u32> {
    let ret = unsafe { v8__Function__GetScriptColumnNumber(self) };
    (ret >= 0).then(|| ret as u32)
  }

  /// Get the (zero-indexed) line number of the function's definition, if available.
  pub fn get_script_line_number(&self) -> Option<u32> {
    let ret = unsafe { v8__Function__GetScriptLineNumber(self) };
    (ret >= 0).then(|| ret as u32)
```
`int` is `std::os::raw::c_int` and shouldn't leak out into the public API when it can be avoided.

I picked `u32` but, depending on what you do with the return value, `usize` might be more appropriate.

**Review by bnoordhuis — APPROVED** [review]
Left some suggestions but mostly LGTM. Thanks for the PR!

File: src/function.rs
Comment (sujayakar) [inline src/function.rs:469]: also woah, TIL about `bool::then` :)

**Comment by bnoordhuis** [comment]
Restarted the CI. Looks like the aarch64 buildbot hit a bug in qemu:
```
(process:17120): GLib-CRITICAL **: 18:55:59.804: g_hash_table_iter_next: assertion 'ri->version == ri->hash_table->version' failed**
ERROR:/build/qemu-KZtMUi/qemu-2.11+dfsg/qom/object.c:1645:object_get_canonical_path_component: code should not be reached
```
Hopefully a one-time thing.

**Comment by bnoordhuis** [comment]
Thanks Sujay, I merged it with a slightly modified commit message (I try to keep the first line <= 50 columns for reasons of legibility in commands like `git log --oneline`.)

**Comment by sujayakar** [comment]
great! could you cut a release at your earliest convenience so I can depend on it rather than my fork?

**Pull request merged**
